### PR TITLE
fix(gatsby-plugin-mdx): Fix graphql query examples in readme

### PR DESCRIPTION
**Pull request by CrashWire**
## Description

https://github.com/gatsbyjs/gatsby/commit/19f7396099c432fee5f58127383a2b87eab755cb breaks the example provided in the readme. This updates the readme to fix it.

### CODE REVIEW

**Comment by mrozilla** [comment]
@CrashWire I was just looking into this! You might want to also update the two other places in the docs where it’s mentioned to make sure the change is complete :)

In your commit, that’s [line 443](https://github.com/gatsbyjs/gatsby/pull/15691/commits/57b24e813d88b20efe788aeb67df151da5162632#diff-bc334e48d29c57bb3c32c0764b5e4242R443) (where it’s consumed from the query) and [line 433](https://github.com/gatsbyjs/gatsby/pull/15691/commits/57b24e813d88b20efe788aeb67df151da5162632#diff-bc334e48d29c57bb3c32c0764b5e4242R433) (where it’s mentioned in the context of additional props)

**Comment by CrashWire** [comment]
> @CrashWire You might want to also update the two other places in the docs where it’s mentioned to make sure the change is complete :)
> 
> In your commit, that’s [line 443](https://github.com/gatsbyjs/gatsby/pull/15691/commits/57b24e813d88b20efe788aeb67df151da5162632#diff-bc334e48d29c57bb3c32c0764b5e4242R443) (where it’s consumed from the query) and [line 433](https://github.com/gatsbyjs/gatsby/pull/15691/commits/57b24e813d88b20efe788aeb67df151da5162632#diff-bc334e48d29c57bb3c32c0764b5e4242R433) (where it’s mentioned in the context of additional props)

Thanks for helping spot the omissions! I’ve made the changes in https://github.com/gatsbyjs/gatsby/pull/15691/commits/35c12f524438b082069e3aed8613131e5228d19a.

File: packages/gatsby-plugin-mdx/README.md [hunk 1]
```diff
@@ -440,17 +440,15 @@ import { MDXRenderer } from "gatsby-plugin-mdx"
 
 export default class MyPageLayout {
   render() {
-    ;<MDXRenderer>{this.props.data.mdx.code.body}</MDXRenderer>
+    ;<MDXRenderer>{this.props.data.mdx.body}</MDXRenderer>
```
Comment (wardpeet) [inline packages/gatsby-plugin-mdx/README.md]: I'm probably wrong but shouldn't this be:
```suggestion
    return (<MDXRenderer>{this.props.data.mdx.body}</MDXRenderer>)
```

**Comment by CrashWire** [comment]
@wardpeet That seems to be right! 

(I was also trying to figure out if @ChristopherBiscardi might have had a reason to include the beginning `;` in https://github.com/gatsbyjs/gatsby/commit/75d49428301383d182767da55593967c5ec5a71c#diff-bc334e48d29c57bb3c32c0764b5e4242L450, but couldn’t think of any.)

**Pull request merged**